### PR TITLE
Specify default option values in readme

### LINKED ISSUE #257: Documenting option defaults
The documentation at https://www.npmjs.com/package/react-tag-input is great at documenting what all the options are.

However it would be really useful if it also documented what the defaults are for each optional option (i.e. if the user doesn't specify it, what is the default behavior?).

### PR DESCRIPTION
**Pull request by tabrath**
Closes #257

### CODE REVIEW

**Comment by ad1992** [comment]
@tabrath can you also add a table in the readme which contains the name of the prop and its default value ? Basically you can convert the list https://github.com/tabrath/react-tags/blob/a148bc8ed2d8ac78b91318db2fd4da2063660c11/README.md#options to table format.

**Comment by tabrath** [comment]
@ad1992 Sure.

File: README.md [hunk 1]
```diff
@@ -130,40 +130,42 @@ Otherwise, you can simply import along with the backend itself (as shown above).
...
-- [`handleInputBlur`](#handleInputBlur)
-- [`minQueryLength`](#minQueryLength)
-- [`removeComponent`](#removeComponent)
-- [`autocomplete`](#autocomplete)
-- [`readOnly`](#readOnly)
-- [`name`](#nameOption)
-- [`id`](#idOption)
-- [`maxLength`](#maxLength)
-- [`inline`](#inline)
+Option | Default
```
Comment (ad1992) [inline README.md]: can you also add `Type` as 2nd column where type of each prop is mentioned ?
Also `description` as 4th column where its mentioned whether the prop is required and the usage of the prop in `1-2` lines ?

**Comment by ad1992** [comment]
@tabrath looks good 👍 . I think you forgot to replace the gif which I mentioned in the last comment. Can you fix it so this can be merged

**Comment by tabrath** [comment]
No, I didn’t forget, just didn’t have time for it, I’ll try making a new gif as soon as possible (busy at work atm). 😊

**Comment by ad1992** [comment]
@tabrath when will you be able to add the GIF ?

**Comment by tabrath** [comment]
@ad1992 Ok, I tried my best :)

**Comment by ad1992** [comment]
Thanks @tabrath

**Pull request merged**